assembly/src/parsers/io_ops/mod.rs

```diff
@@ -257,7 +257,7 @@ pub fn parse_storew(
 ///   values (represented by 32-bit limbs), divides one value by another, and injects the quotient
 ///   and the remainder into the advice tape.
 pub fn parse_adv_inject(
-    span_ops: &mut Vec<Operation>,
+    span_ops: &mut [Operation],
```

Comment:
I think the comment above needs to be updated as well - but could be done in a different PR (maybe just leave a TODO for now?).

Reply:
I don't understand what's needed here - I was just fixing a clippy warning. Is the issue with the comment that the code is actually appending to the `decorators` rather than to `span_ops`? I'm not going to leave a TODO, as I'm not sure what to put, but I'm happy to open an issue once I understand what's needed. For now I'm going to merge this PR so I can move forward to wrap up the b_chip column.

Reply:
Sounds good! Yes, the changes are that we are appending to `decorators`, and actually `span_ops` does not need to be mutable here.